### PR TITLE
ballerina 0.980.1

### PR DESCRIPTION
**Pull request by rasika**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by ilovezfs — CHANGES_REQUESTED** [review]
Please revert the formula to using libexec as we don't want to just install things in `prefix` like this.

File: Formula/ballerina.rb [hunk 1]
```diff
@@ -28,6 +28,8 @@ def install
 
     bin.install "bin/ballerina", "bin/composer"
     libexec.install Dir["*"]
+    #link tooling-plugin jar for the vscode
+    lib.install_symlink libexec/"lib/resources" => "resources"
```
Comment (ilovezfs) [inline Formula/ballerina.rb]: The `=> "foo"` is only needed if the name is different.

**Comment by rasika** [comment]
Thanks for the review. Changes applied to facilitate below;

Once ballerina installed through homebrew; users should be able to point **ballerina-home** path in their VSCode ballerina plugin. This fails when `lib/resources` folder is absent in `/usr/local/Cellar/ballerina/0.980.1`.

screen shot 2018-07-24 at 1 39 41 pm

How about adding a symlink for the `lib/resources` folder?

File: Formula/ballerina.rb [hunk 2]
```diff
@@ -28,6 +28,8 @@ def install
 
     bin.install "bin/ballerina", "bin/composer"
     libexec.install Dir["*"]
+    #link tooling-plugin jar for the vscode
```
Comment (ilovezfs) [inline Formula/ballerina.rb]: Can you please explain this? CC @DomT4

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/ballerina.rb (hunk 2 above)
Comment (rasika) [reply Formula/ballerina.rb]: Can remove that comment. Added below;
`lib.install_symlink libexec/"lib/resources" => "resources"` to create a symlink for the resources folder.

Comment (DomT4) [reply Formula/ballerina.rb]: I don't think it'd be especially difficult for [upstream](https://github.com/ballerina-platform/ballerina-lang/tree/master/tool-plugins/vscode) to add a `ballerina.resources` variable to resolve this? Might be worth asking them, but this is harmless enough that I don't have an objection to it personally.

Comment (rasika) [reply Formula/ballerina.rb]: +1 for **symlink** since introducing yet another `ballerina.resources` is an overkill for the end user.

**Review by ilovezfs — APPROVED** [review]

**Pull request merged**